Gemfile

```diff
@@ -69,7 +69,7 @@ gem 'browser'
 gem 'haml-rails'
 
 # intercepts outgoing emails in non-production environment
-gem 'mail_interceptor', github: 'bigbinary/mail_interceptor', group: [:development, :staging]
+gem 'mail_interceptor', github: 'bigbinary/mail_interceptor', group: [:development, :staging, :test]
```

Comment:
No need to intercept in test. Then in your tests the recipient will change to the intercepted email.